packages/search-service/src/e2e/search.spec.ts

```diff
@@ -1,16 +1,44 @@
-import dotenv from 'dotenv';
+import dotenv from "dotenv";
```

Comment:
Can you change all the double-quotes to single? To make it more consistent with the rest of the formatting in the repo.
Also make sure your editor uses the vscode settings specified in the `.vscode/settings.json` file at the root of the repo. It already has all the necessary formatting rules mentioned.

Reply:
Updated